lib/nerve/service_watcher.rb

```diff
@@ -25,6 +26,11 @@ def initialize(service={})
       # configure the reporter, which we use for reporting status to the registry
       @reporter = Reporter.new_from_service(service)
 
+      # configure the rate limiter for updates to the reporter
+      @rate_limiter = RateLimiter.new(average_rate: service['rate_limit_avg'] || 10,
```

Comment:
can we support no rate limiting by default? we can get rid of rate_limit_enabled

Reply:
Do you mean instead of a "shadow mode", we only create the rate limiter if it is enabled?
That would mean no metrics or logs until we enable it, however